src/install/python/python-frameworks/python-agent-web2py.mdx

```diff
@@ -1,22 +1,12 @@
 ---
-title: Python agent and Web2py web framework
-tags:
-  - Agents
-  - Python agent
-  - Web frameworks and servers
-metaDescription: Details on using the Python agent with an application that uses the Web2py web framework.
-redirects:
-  - /docs/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
-  - /docs/python/python-agent-and-web2py
-  - /docs/agents/python-agent/web-frameworks/python-agent-and-web2py
-  - /docs/agents/python-agent/web-frameworks/python-agent-web2py
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using Web2Py
 ---
 
-Web2py is an open source framework for creating web-based Python applications. You can integrate the Python agent with your Web2py app to monitor your app's performance. For general install procedures, see [Install the Python agent](/docs/agents/python-agent/installation-configuration/python-agent-installation).
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command.
 
-## Integrate with Web2py [#web2py-integration]
-
-Choose the Web2py set-up you use from the list below for more information:
+Choose the Web2py setup you use from the list below:
 
 <CollapserGroup>
```

Comment:
Keep the runweb2py section here. Perhaps outside of a collapser so it matches the other pages. and remove the other sections as they are just pointing to the server instructions.

Reply:
@hmstepanek I removed the following in the collapser:

* running-with-mod_wsgi
* running-with-gunicorn
* running-with-uwsgi

I moved the Web2Py information out of the collapser to match the other sections.